Makefile

```
@@ -15,6 +15,12 @@ bench-cairo-m:
 	cd cairo-m && \
 	RUSTFLAGS="-C target-cpu=native" cargo run --release
 
+bench-noir-provekit:
+	cd noir_provekit/runner/test_data && \
+	nargo build && \
```

`nargo build` and `nargo compile` seems to do the exact same thing, however only `nargo compile` is displayed in the `nargo --help` so I'd rather use it.

```suggestion
	nargo compile && \
```